test/api-callbacks.js

```diff
@@ -39,18 +39,12 @@ var auctionExample = require('./fixture/auctionexample1');
 
 describe('Api Callbacks - Tests', function() {
 
-  function clearEvents() {
-    Event.removeAllListeners();
-    Event.observeImmediate_ = null;
-    Event.observeImmediate_ = {};
-  }
-
   beforeEach(function() {
-    clearEvents();
+    Event.removeAllListeners();
   });
 
   afterEach(function() {
```

Comment:
Did you mean that neither `beforeEach` or `afterEach` ?

w/o `beforeEach`, `Event.removeAllListeners()` some tests are not run and this looks like an issue with the test code.

Looking at this.